optin/reporting/test/models/runTotalRecordTests.js

```diff
@@ -0,0 +1,13 @@
+const { expect } = require('chai');
```

Comment:
since this is shared set of test(s), please move to test/test folder to not intermix with tests like we do in other places (e.g. sdk/javascript).

Reply:
likewise should move similar run* files